### DISCUSSION TITLE
Beginner problem

### DISCUSSION THREAD

**Discussion by petrarislin** [TumblThreeApp/TumblThree#371] [Q&A (Question & Answer)]
Tumblthree 2.8.3.357 under win10, not succeeding at any tumblr blog. I can add the blog to the main window, but nothing happens after clicking Download. I have authenticated successfully under Settings | Connection. The tumblthree.log shows  the program starting, but nothing else. Am I missing a required framework or something? For example, https://babyanimalgifs.tumblr.com. Blog type is "Tumblr", blog is not NSFW. Thanks.

**Comment by thomas694** [comment]
> - To start the download process, click on 'Download'. The application will regularly check for (new) blogs in the queue and start processing them, until you stop the application by pressing 'Stop'. So, you can either add blogs to the queue via 'Add to Queue' or double click first and then click 'Download', or you start the download process first and add blogs to the queue afterwards.
